ospd/vts.py

```diff
@@ -45,6 +45,14 @@ def __init__(
     def __contains__(self, key: str) -> bool:
         return key in self._vts
 
+    def __iter__(self) -> Iterator[str]:
+        if hasattr(self.vts, '__iter__'):
+            return self.vts.__iter__()
+
+        # Use iter because python3.5 has no support for
+        # iteration over DictProxy.
+        return iter(self.vts.keys())
+
```

Comment:
After running a scan, it seems we need also to implement __getitem__(). 
```
def __getitem__(self, key):
    return self.vts.__getitem__(key)
```

Reply:
I guess some ospd-openvas is using vts[vt_oid]? IMHO we should replace this code by vts.get(vt_oid) as I already did in ospd.

Reply:
I've added __getitem__ for compatibility.